music_assistant/providers/nicovideo/helpers/hls_seek_optimizer.py

```diff
@@ -1,12 +1,19 @@
-"""HLS seek optimizer for nicovideo provider."""
+"""HLS seek optimizer for nicovideo provider.
+
+This module implements a workaround for FFmpeg's seeking limitations with fragmented MP4
+HLS playlists (see https://trac.ffmpeg.org/ticket/7359).
+
+NOTE: This entire module can be removed once Music Assistant requires FFmpeg 8.0+,
+      which fixes the input-side -ss seeking issue (commit 380a518c, 2024-11-10).
```

Comment:
I have no problem switching to ffmpeg 8.0 - in the end we build our own ffmpeg version in the base image.
Maybe a good one to do after 2.7 is released

Reply:
please remember us to do the ffmpeg bump after 2.7 stable release or simply open the PR yourself then.
Dockerfile.base contains the ffmpeg builder